api/stream-relay/index.js

```diff
@@ -488,10 +510,29 @@ export const wrap = dgram => {
      * @ignore
      */
     send (data, port, address, socket = this.socket) {
+      this.sendQueue.push({ data, port, address, socket })
+      this._scheduleSend()
+    }
+
+    _scheduleSend () {
```

Comment:
we can add `@private` JSDoc tag here to improve TS definitions a bit, same for `_dequeue`